test/team_collection_test.rb

```diff
@@ -0,0 +1,22 @@
+require './test/test_helper'
+require './lib/team'
+require './lib/team_collection'
+
+class TeamsCollectionTest < Minitest::Test
+
+  def setup
+    @total_teams = TeamCollection.new("./data/teams_sample.csv")
+  end
+
+  def test_it_exists
+    assert_instance_of TeamCollection, @total_teams
+  end
+
+  def test_it_has_total_teams
+    @total_teams.create_games('./data/teams_sample.csv')
+    assert_equal 3, @total_teams.total_teams.length
+  end
+
+  def test_it_has_attributes
```

Comment:
I don't think we will need to test for attributes for team collection.

Reply:
Oh oops - you're right. I had started this test in the wrong file and moved it to team_test. Not sure how I missed this. I'll recommit right now so the updated version gets merged :)